api/integrations/github/client.py

```diff
@@ -12,3 +45,66 @@ def generate_token(installation_id: str, app_id: int) -> str:
     Github(auth=auth)
     token = auth.token
     return token
+
+
+def generate_jwt_token(app_id: int) -> str:  # pragma: no cover
+    github_auth: Auth.AppAuth = Auth.AppAuth(
+        app_id=app_id,
+        private_key=settings.GITHUB_PEM,
+    )
+    token = github_auth.create_jwt()
+    return token
+
+
+def post_comment_to_github(
+    installation_id: str, owner: str, repo: str, issue: str, body: str
+) -> typing.Dict[str, typing.Any]:
+    token = generate_token(
+        installation_id,
+        settings.GITHUB_APP_ID,
+    )
+
+    url = f"{GITHUB_API_URL}repos/{owner}/{repo}/issues/{issue}/comments"
+    headers = {
+        "Accept": "application/vnd.github.v3+json",
+        "Authorization": f"Bearer {token}",
+    }
+
+    payload = {"body": body}
+    response = response = requests.post(url, json=payload, headers=headers, timeout=10)
+    response.raise_for_status()
+
+    return response.json()
+
+
+def delete_github_installation(installation_id: str) -> requests.Response:
+    url = f"{GITHUB_API_URL}app/installations/{installation_id}"
+    headers = build_request_headers(installation_id, useJWT=True)
+    response = requests.delete(url, headers=headers, timeout=10)
+    return response
```

Comment:
Why no `raise_for_status()` here?

Reply:
Covered in the view

Reply:
Is there a reason to handle the status code in the view specifically?

Reply:
Being a class View it is just easier and clearer to me.